main/User_config.h

```diff
@@ -497,7 +497,132 @@ int lowpowermode = DEFAULT_LOW_POWER_MODE;
+#    define ANEOPIX_CRITICAL    ANEOPIX_BLACK // unused
+#    define ANEOPIX_POWER       ANEOPIX_BLACK // unused
+#    define ANEOPIX_BOOT        ANEOPIX_AQUA
+#    define ANEOPIX_OFF         ANEOPIX_BLACK
+#  else
+#    define ANEOPIX_INFO        ANEOPIX_BLUE_DIM // dimmed blue info background
+#    define ANEOPIX_ERROR       ANEOPIX_RED_DIM
+#    define ANEOPIX_SENDRECEIVE ANEOPIX_GOLD // bright gold  = sending
+#    define ANEOPIX_CRITICAL    ANEOPIX_BLACK // unused
+#    define ANEOPIX_POWER       ANEOPIX_BLACK // unused
+#    define ANEOPIX_BOOT        ANEOPIX_AQUA
+#    define ANEOPIX_OFF         ANEOPIX_BLACK
+#  endif
+#  ifndef ANEOPIX_IND_DATA_GPIO2
+// during boot the RGB LED is on to signal also reboots
+#    define SetupIndicators()                             \
+      pinMode(RGB_LED_POWER, OUTPUT);                     \
+      digitalWrite(RGB_LED_POWER, HIGH);                  \
+      leds.begin();                                       \
+      leds.setPixelColor(ANEOPIX_INFO_LED, ANEOPIX_BOOT); \
+      leds.show();
+#  else
+#    define SetupIndicators()            \
+      pinMode(RGB_LED_POWER, OUTPUT);    \
+      digitalWrite(RGB_LED_POWER, HIGH); \
+      leds.begin();                      \
+      leds2.begin();
+#  endif
+#  define ErrorIndicatorON()                              \
+    leds.setPixelColor(ANEOPIX_ERROR_LED, ANEOPIX_ERROR); \
+    leds.show();
+#  define ErrorIndicatorOFF()                           \
+    leds.setPixelColor(ANEOPIX_ERROR_LED, ANEOPIX_OFF); \
+    leds.show();
+#  define SendReceiveIndicatorON()                                      \
+    leds.setPixelColor(ANEOPIX_SEND_RECEIVCE_LED, ANEOPIX_SENDRECEIVE); \
+    leds.show();
+#  define SendReceiveIndicatorOFF()                             \
+    leds.setPixelColor(ANEOPIX_SEND_RECEIVCE_LED, ANEOPIX_OFF); \
+    leds.show();
+#  define InfoIndicatorON()                             \
+    leds.setPixelColor(ANEOPIX_INFO_LED, ANEOPIX_INFO); \
+    leds.show();
+#  define InfoIndicatorOFF()                           \
+    leds.setPixelColor(ANEOPIX_INFO_LED, ANEOPIX_OFF); \
+    leds.show();
+#  ifdef ANEOPIX_IND_DATA_GPIO2 // Used for relay power indicator
+// For the critical ON indicator there is no method to turn it off, the only way is to unplug the device
+// This enable to have persistence of the indicator to inform the user
+#    define CriticalIndicatorON()                             \
+      leds.setPixelColor(ANEOPIX_INFO_LED, ANEOPIX_CRITICAL); \
+      leds.show();
+#    define PowerIndicatorON()                             \
+      leds2.setPixelColor(ANEOPIX_INFO_LED, ANEOPIX_INFO); \
+      leds2.show();
+#    define PowerIndicatorOFF()                           \
+      leds2.setPixelColor(ANEOPIX_INFO_LED, ANEOPIX_OFF); \
+      leds2.show();
+#  endif
+#  define SetupIndicatorInfo()
+#  define SetupIndicatorSendReceive()
+#  define SetupIndicatorError()
+#else // FastLED remains standard for RGB_INDICATORS
```

Comment:
I think we could completely replace the use of FASTLED for indicators, and remove the code below, thoughts ?

Reply:
I think so too! I was just afraid of a big step. It would require a rather simple find/replace action in environments.ini. Shall I make a suggestion?

Reply:
Yes go ahead